test/end-to-end/vehicle.js

```diff
@@ -457,6 +461,20 @@ test('vehicle request - override auth header', async(t) => {
   });
 });
 
+test('vehicle request - get charge limit', async(t) => {
+  const response = await t.context.kia.getChargeLimit();
+
+  t.is(typeof response.limit, 'number');
+});
+
+test('vehicle request - set charge limit', async(t) => {
+  await t.context.kia.setChargeLimit(0.9);
+
+  const response = await t.context.kia.getChargeLimit();
```

Comment:
Really clever, but this won't work. the test vehicle doesn't have any state. In other words, `getChargeLimit` is just going to return something random.

Reply:
What do you think would be an appropriate test for this? I'm thinking of just checking the response of `setChargeLimit`